pkg/systemd/services/service-details.jsx

```diff
@@ -576,3 +655,26 @@ ServiceDetails.propTypes = {
     permitted: PropTypes.bool.isRequired,
     isValid: PropTypes.func.isRequired,
 };
+
+const DeleteModal = (props) => {
```

Comment:
Please expand `props` here, i.e. `({ handleCancel, handleDelete })`. That makes it easier to see which props a component expects, and enables compile-time type/variable checks.